modules/ossm-updating-smcp.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/customizing-installation-ossm.adoc
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+[id="ossm-updating-smcp_{context}"]
+= Updating the ServiceMeshControlPlane
+
+Configure {ProductName} to work with your app by creating and configuring a `ServiceMeshControlPlane`. The `ServiceMeshControlPlane` resource defines the configuration to be used during installation. You can deploy the default configuration provided by Red Hat or customize the `ServiceMeshControlPlane` resource for your microservices and workflows.
+
+[id="ossm-control-plane-deploy-operatorhub_{context}"]
+== Editing the `ServiceMeshControlPlane` from the web console
+
+Follow this procedure to edit the `ServiceMeshControlPlane` with the {product-title} web console.
+
+.Procedure
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Select the project where you installed the control plane, for example istio-system, from the Project menu.
+
+. Click the {ProductName} Operator. In the *Istio Service Mesh Control Plane* column, click *Create ServiceMeshControlPlane*.
+
+. On the *Create Service Mesh Control Plane* page, modify the YAML for the default `ServiceMeshControlPlane` template.
+
+[id="ossm-control-plane-deploy-cli_{context}"]
+== Editing the `ServiceMeshControlPlane` from the CLI
+
+Follow this procedure to create or edit the `ServiceMeshControlPlane` with the command line.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. Enter the following command. Then, enter your username and password when prompted.
++
+[source,terminal]
+----
+$ oc login https://{HOSTNAME}:6443
+----
++
+. Change to the project where you installed the control plane, for example istio-system.
++
+[source,terminal]
+----
+$ oc project istio-system
+----
++
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml`.
```

Comment:
The heading for this topic is "Editing" not "Creating", so why are we creating a file?  Shouldn't they be opening an existing file?